plugins/available/nvm.plugin.bash

```diff
@@ -6,7 +6,13 @@ cite about-plugin
 about-plugin 'node version manager configuration'
 
 export NVM_DIR="$HOME/.nvm"
```

Comment:
What about this variable? When the brew nvm is used, it doesn't make any sense to set this variable to `$HOME/.nvm`, right? Wouldn't it be better to move this into the respective `if` branches?